exporter/prometheusexporter/README.md

```diff
@@ -1,14 +1,14 @@
 # Prometheus Exporter
 
-Exports data to a [Prometheus](https://prometheus.io/) back-end.
+Exports data in a [Prometheus format](https://prometheus.io/docs/concepts/data_model/). Allows scraping them by the [Prometheus](https://prometheus.io/) back-end.
 
 Supported pipeline types: metrics
 
 ## Getting Started
 
 The following settings are required:
 
-- `endpoint` (no default): the address on which the Prometheus scrape handler will be run on.
+- `endpoint` (no default): the address on which data will be exposed and against which the Prometheus scrape handler will be run on.
```

Comment:
```suggestion
- `endpoint` (no default): the address on which metrics will be exposed by the Prometheus scrape handler.
```
"prometheus scrape handler" is a reference to the http "handler" in this exporter, rather than referring to the prom server.

Reply:
Ah, makes sense, thanks!